src/main/java/dinkplugin/DinkPluginConfig.java

```diff
@@ -698,11 +698,24 @@ default boolean deathEmbedKeptItems() {
         return false;
     }
 
+    @ConfigItem(
+        keyName = "deathSkipSafe",
+        name = "Ignore Safe Deaths",
+        description = "Whether deaths in safe areas should be ignored.<br/>" +
+            "Note: Inferno and TzHaar Fight Cave deaths are still sent with this setting enabled",
```

Comment:
Given this checkbox exists now, do we still need to treat these 2 activities as exceptions?